src/table/Root.jsx

```diff
@@ -8,17 +8,18 @@ import { TableContextProvider } from './context';
 export function render(reactRoot, props) {
   const { muiTheme, direction, selectionsAPI } = props;
 
-  reactRoot.render(
-    <React.StrictMode>
-      <StyleSheetManager stylisPlugins={direction === 'rtl' ? [rtlPluginSc] : []}>
-        <ThemeProvider theme={muiTheme}>
-          <TableContextProvider selectionsAPI={selectionsAPI}>
-            <TableWrapper {...props} />
-          </TableContextProvider>
-        </ThemeProvider>
-      </StyleSheetManager>
-    </React.StrictMode>
-  );
+  reactRoot &&
+    reactRoot.render(
```

Comment:
```suggestion
    reactRoot?.render(
```

Reply:
I had a test, and found that `reactRoot?.render` did not work as expected. So I switched to `reactRoot && reactRoot.render()`. But I will try it again.

Reply:
Updated